src/workers/airtable-sync/paymentWorker.js

```diff
@@ -9,12 +9,17 @@ const {
   paymentRequestsFields,
   paymentRequestsSensitiveFields,
 } = require("~airtable/tables/paymentRequests");
+const sendErrorNotification = require("~slack/errorNotification");
 const newExternalDonorPayment = require("./actions/payments/newExternalDonorPayment");
 const newPaymentRequest = require("./actions/payments/newPaymentRequest");
 const updateReimbursementMessage = require("./actions/payments/updateReimbursementStatus");
 
 const defaultInterval = 10000;
 
+const errFunc = (error) => {
+  sendErrorNotification(error);
```

Comment:
optional nit: can you make the function async and await the sendError? functionally the same, but a bit more resistant to a potential weird future change-detector update